package.json

```diff
@@ -32,33 +32,34 @@
         ]
     },
     "scripts": {
-        "lint:sandbox": "eslint src/sandbox/sandbox.default.ts",
+        "lint:sandbox": "eslint src/sandbox/sandbox.default.tsx",
         "lint:code": "eslint --ext .ts,.js",
         "lint:fix": "yarn lint:code --fix",
         "lint:markdown": "node src/scripts/markdown-lint.js",
         "lint": "yarn lint:code . && yarn lint:markdown",
         "script:copy": "node ./dev/scripts/copy.js",
         "script:extract-rules-meta-info-to-google-sheets": "node dist/scripts/extract-rules-meta-info-to-google-sheets.js",
+        "script:update-deps-local": "node node_modules/@eigenspace/helper-scripts/scripts/project/update-deps.js",
         "compile": "tsc",
         "build": "yarn script:copy && yarn compile",
-        "deploy:publish": "node node_modules/@eigenspace/helper-scripts/scripts/ci/publish.js"
+        "deploy:publish": "node node_modules/@eigenspace/helper-scripts/scripts/ci/publish.js",
+        "deploy:publish-subprojects": "node node_modules/@eigenspace/helper-scripts/scripts/ci/publish.js --projectPaths[]=/,/src/packages/eslint-plugin-eigenspace-react,/src/packages/eslint-plugin-eigenspace-redux,/src/packages/eslint-plugin-eigenspace-script"
```

Comment:
I would leave the npm script interface consistent for other projects. Would leave just `deploy: publish`